I/6663: Updated keyboard support documentation - added table navigation and widget keystrokes. (#6697)
ckeditor/ckeditor5: docs/features/keyboard-support.md

			<td colspan="2"><kbd>Tab</kbd></td>
		</tr>
		<tr>

> 			<th colspan="3">When a widget is selected (e.g., image, table, horizontal line, etc.)</th>

Reviewer: ```suggestion
			<th colspan="3">When a widget is selected (for example: image, table, horizontal line, etc.)</th>
```

		</tr>
		<tr>

> 			<td>Insert paragraph directly after widget</td>

Reviewer: ```suggestion
            <td>Insert a new paragraph directly after a widget</td>
```

			<td colspan="2"><kbd>Enter</kbd></td>
		</tr> 
		<tr>

> 			<td>Insert new paragraph directly before widget</td>

Reviewer: ```suggestion
			<td>Insert a new paragraph directly before a widget</td>
```

			<td colspan="2"><kbd>Shift</kbd> + <kbd>Enter</kbd></td>
		</tr>
		<tr>
			<th colspan="3">In a table cell</th>
		</tr>
		<tr>

> 			<td>Move selection to the next cell</td>

Reviewer: ```suggestion
			<td>Move the selection to the next cell</td>
```

			<td colspan="2"><kbd>Tab</kbd></td>
		</tr>
		<tr>

> 			<td>Move selection to the previous cell</td>

Reviewer: ```suggestion
			<td>Move the selection to the previous cell</td>
```
